shell/platform/windows/keyboard_key_handler.cc

```diff
@@ -48,10 +47,10 @@ void KeyboardKeyHandler::KeyboardHook(int key,
   incoming->callback = std::move(callback);
 
   if (pending_responds_.size() > kMaxPendingEvents) {
-    std::cerr
+    FML_LOG(ERROR)
```

Comment:
Should this be a warning? Same comment for similar log in `shell/platform/windows/keyboard_manager.cc`

Reply:
Good question. If there are more responses than pending events, it suggests an invariant was violated. Error seems pretty reasonable to me.